--- fuel-library/utils/jenkins/fuel_noop_tests.rb	2015-11-09 19:51:53.000000000 +0000
+++ fuel-library/utils/jenkins/fuel_noop_tests.rb	2015-11-09 19:51:17.000000000 +0000
@@ -271,8 +271,10 @@
   # @return [Array<TrueClass,FalseClass,NilClass>] success and empty report array
   def self.rspec(spec)
     inside_noop_tests_directory do
+      identity = spec.split('/')[-1]
+      ENV["SPEC"] = identity
       command = "rspec #{RSPEC_OPTIONS} #{spec}"
-      command = 'bundle exec ' + command if options[:bundle]
+      command = "bundle exec " + command if options[:bundle]
       if options[:filter_examples]
         options[:filter_examples].each do |example|
           command = command + " -e #{example}"
